spec/factories/conversation.rb

```diff
@@ -1,4 +1,39 @@
 FactoryBot.define do
   factory :conversation do
+    factory :convarsation_with_users do
```

Comment:
typo error here `conversation`:
`conversation_with_users`

check the other cases too like `conversation_with_users_and_messages`

Reply:
You can also use a trait for these cases:
```
FactoryBot.define do
  factory :conversation do
    trait :with_users do
    end
    trait :with_users_and_messages do
    end
  end
end
```

And then you call the same factory but with the trait as parameter:
`let!(:conversation) { create(:convarsation, :with_users, user1: user1, user2: user2) }`

You can check how traits work here:
[Traits](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md)